src/components/btns.js

```diff
@@ -0,0 +1,120 @@
+import { v4 as uuidv4 } from 'uuid';
+
+const buttons = [
+  {
+    symbol: 'ACD',
+    id: uuidv4(),
```

Comment:
- [ ] Kindly make this symbol 'AC' so that it can actually clear the field now it just adds ACD to the result.
![image](https://user-images.githubusercontent.com/59416136/176731876-958f6844-a83b-47b1-acd7-9be74f708d48.png)

Reply:
This issue is well resolved! ✔️